src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/operations/abstractOperationDetails.ts

```diff
@@ -56,9 +56,27 @@ abstract class abstractOperationDetails extends dialogViewModelBase {
                 }
             });
     }
+    
+    protected calculateProcessingSpeed(processed: number, name?: string): number {
+        let durationInSeconds;
+        
+        switch (name) {
+            case 'Documents': {
```

Comment:
we don't use { and around case statements IIRC + use double quotes whenever possible
```
case "Documents:
    durationInSeconds = this.op.durationInSecondsDocuments();
    break;

```
etc.

Reply:
right - done